src/core/class_db.cpp

```diff
@@ -352,8 +352,9 @@ void ClassDB::initialize(GDExtensionInitializationLevel p_level) {
 }
 
 void ClassDB::deinitialize(GDExtensionInitializationLevel p_level) {
+	std::set<StringName> to_erase;
 	for (std::vector<StringName>::reverse_iterator i = class_register_order.rbegin(); i != class_register_order.rend(); ++i) {
-		const StringName &name = *i;
+		const StringName& name = *i;
```

Comment:
This seems like it's switching away from the usual Godot style? The CI should tell us in a few minutes, though :-)

Reply:
Yeah it's vscode for some reason it'doesnt apply the formatting automatically when I save, not sure why but it's my setup's fault probably, sorry for the noise 🥲